### PR TITLE
To automate creation of ADF data pipeline, activities and databricks job schedue

### PR DESCRIPTION
**Pull request by jedheaj314**
Includes fix for deploy (make command).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,18 +54,23 @@ deploy: bootstrap ## Deploy all infrastructure
 	&& cd ${MAKEFILE_DIR}/infrastructure \
 	&& terragrunt run-all apply --terragrunt-non-interactive
 
-deploy-core: bootstrap ## Deploy core infrastructure
```
Comment (tanya-borisova) [inline Makefile:57]: We don't need to make `deploy-core` depend on `build-transform-library`, but `deploy` does need to depend on `build-transform-library`

File: infrastructure/transform/locals.tf [hunk 2]
```diff
@@ -1,3 +1,12 @@
 locals {
-  storage_account_name = "dbfs${var.truncated_naming_suffix}"
+  storage_account_name    = "dbfs${var.truncated_naming_suffix}"
+  adb_linked_service_name = "ADBLinkedServiceViaMSI"
+
+  python_file_local_path = "../../transform/features"
+  python_file_dbfs_path  = "dbfs:"
```
Comment (tanya-borisova) [inline infrastructure/transform/locals.tf:6]: Sorry just seen this, but this seems the same as `whl_file_dbfs_path`, and also I guess it's not a path? How about calling it `dbfs_prefix` and removing `whl_file_dbfs_path`

**Review by jjgriff93 — APPROVED** [review]

File: infrastructure/transform/locals.tf (hunk 2 above)
Comment (tanya-borisova) [reply infrastructure/transform/locals.tf:6]: Will resolve just to get it merged as this will be superseded by some of my work anyway

**Review by tanya-borisova — APPROVED** [review]

**Pull request merged**